pkg/image/prune/imagepruner.go

```diff
@@ -491,7 +491,15 @@ func (p *imagePruner) Run(pruneImage ImagePruneFunc, pruneStream ImageStreamPrun
 	pruneImages(p.g, imageNodes, pruneImage, pruneStream, pruneManifest)
 
 	layerNodes := layerNodeSubgraph(allNodes)
-	pruneLayers(p.g, layerNodes, pruneLayer, pruneBlob)
+	registryURL := ""
+	if len(imageNodes) > 0 {
+		if ref, err := imageapi.ParseDockerImageReference(imageNodes[0].Image.DockerImageReference); err == nil {
+			registryURL = ref.Registry
+		} else {
+			glog.Errorf("error determining registry URL from %q: %v", imageNodes[0].Image.DockerImageReference, err)
```

Comment:
@smarterclayton 

> Is this not a failure condition? Or do we fail safe?

We should never hit this, but just in case, I added https://github.com/openshift/origin/pull/3515/files#diff-fe0c15d773fd6e1d15dfbb49cbc95c4aR549 as a fail safe.